src/nni_manager/rest_server/restHandler.ts

```diff
@@ -268,6 +269,19 @@ class NNIRestHandler {
         });
     }
 
+    private getTrialLog(router: Router): void {
+        router.get('/trial-log/:id/:type', async(req: Request, res: Response) => {
+            this.nniManager.getTrialLog(req.params.id, req.params.type).then((log: string) => {
+                if (log === '') {
```

Comment:
I don't suggest sending unstructured infomation through REST API.
Either leave it empty and let web UI to check it, or encode it as a JSON object.

The idea behind is that REST API is designed for sending "raw data" which can be easily understood by program, rather than "UI string".

Reply:
I think here `"No logs available"` is also a "raw data", since this logic only check if the return value of `getTrialLog` is an empty string and if so, change it to `No logs available`.